terraform/010-cluster/vars.tf

```diff
@@ -68,6 +74,24 @@ variable "tags" {
   default     = {}
 }
 
+variable "transit_gateway_id" {
+  description = "The ID of the transit gateway to attach to when using create_transit_gateway_attachment."
```

Comment:
Was this `create_transit_gateway_attachment` renamed? I don't see it used anywhere else yet.

Reply:
Thanks, yes, it looks like that variable was renamed.